python/tests/test_lowlevel.py

```diff
@@ -4402,8 +4402,8 @@ def test_c_tsk_err_node_out_of_bounds(self, bad_node):
     def test_c_tsk_err_bad_windows(self):
         ts = self.example_ts()
         L = ts.get_sequence_length()
-        with pytest.raises(_tskit.LibraryError, match="BAD_WINDOWS"):
-            self.pair_coalescence_counts(ts, windows=[-1.0, L])
+        with pytest.raises(tskit.LibraryError, match="TSK_ERR_BAD_WINDOWS"):
```

Comment:
should all these be `_tskit.LibraryError`?

Reply:
what's the distinction?

Reply:
I don't think it matters, it's probably the same exception. The _tskit module is the low-level C Python module, which defines LibraryError. Somewhere in Python we're importing this exception from _tskit, which then becomes tskit.LibraryError also.

So I guess Peter's right, it's better to directly catch the exception that we know the C code will throw.

Reply:
shall I go ahead and make this change across the board in `tests/test_lowlevel.py`, then? there's a number of places (outside of the pair coalescence stuff) where `tskit.LibraryError` is checked for